po/el.po

```diff
@@ -55,3 +55,242 @@ msgstr ""
+
+#: ../vnc.html:97
+msgid "Middle mousebutton"
+msgstr "Μεσαίο Πλήκτρο Ποντικιού"
+
+#: ../vnc.html:100
+msgid "Right mousebutton"
+msgstr "Δεξί Πλήκτρο Ποντικιού"
+
+#: ../vnc.html:103
+msgid "Keyboard"
+msgstr "Πληκτρολόγιο"
+
+#: ../vnc.html:103
+msgid "Show Keyboard"
+msgstr "Εμφάνιση Πληκτρολογίου"
+
+#: ../vnc.html:110
+msgid "Extra keys"
+msgstr "Επιπλέον πλήκτρα"
+
+#: ../vnc.html:110
+msgid "Show Extra Keys"
+msgstr "Εμφάνιση Επιπλέον Πλήκτρων"
+
+#: ../vnc.html:115
+msgid "Ctrl"
+msgstr "Ctrl"
+
+#: ../vnc.html:115
+msgid "Toggle Ctrl"
+msgstr "Εναλλαγή Ctrl"
+
+#: ../vnc.html:118
+msgid "Alt"
+msgstr "Alt"
+
+#: ../vnc.html:118
+msgid "Toggle Alt"
+msgstr "Εναλλαγή Alt"
+
+#: ../vnc.html:121
+msgid "Send Tab"
+msgstr "Αποστολή Tab"
+
+#: ../vnc.html:121
+msgid "Tab"
+msgstr "Tab"
+
+#: ../vnc.html:124
+msgid "Esc"
+msgstr "Esc"
+
+#: ../vnc.html:124
+msgid "Send Escape"
+msgstr "Αποστολή Escape"
+
+#: ../vnc.html:127
+msgid "Ctrl+Alt+Del"
+msgstr "Ctrl+Alt+Del"
+
+#: ../vnc.html:127
+msgid "Send Ctrl-Alt-Del"
+msgstr "Αποστολή Ctrl-Alt-Del"
+
+#: ../vnc.html:135
+msgid "Shutdown/Reboot"
+msgstr "Κλείσιμο/Επανεκκίνηση"
+
+#: ../vnc.html:135
+msgid "Shutdown/Reboot..."
+msgstr "Κλείσιμο/Επανεκκίνηση..."
+
+#: ../vnc.html:141
+msgid "Power"
+msgstr "Απενεργοποίηση"
+
+#: ../vnc.html:143
+msgid "Shutdown"
+msgstr "Κλείσιμο"
+
+#: ../vnc.html:144
+msgid "Reboot"
+msgstr "Επανεκκίνηση"
+
+#: ../vnc.html:145
+msgid "Reset"
+msgstr "Επαναφορά"
+
+#: ../vnc.html:150 ../vnc.html:156
+msgid "Clipboard"
+msgstr "Πρόχειρο"
+
+#: ../vnc.html:160
+msgid "Clear"
+msgstr "Καθάρισμα"
+
+#: ../vnc.html:166
+msgid "Fullscreen"
+msgstr "Πλήρης Οθόνη"
+
+#: ../vnc.html:171 ../vnc.html:178
+msgid "Settings"
+msgstr "Ρυθμίσεις"
+
+#: ../vnc.html:181
+msgid "Encrypt"
+msgstr "Κρυπτογράφηση"
+
+#: ../vnc.html:184
+msgid "True Color"
+msgstr "Πραγματικά Χρώματα"
+
+#: ../vnc.html:187
+msgid "Local Cursor"
+msgstr "Τοπικός Δρομέας"
+
+#: ../vnc.html:190
+msgid "Clip to Window"
+msgstr "Αποκοπή στο όριο του Παράθυρου"
+
+#: ../vnc.html:193
+msgid "Shared Mode"
+msgstr "Κοινόχρηστη Λειτουργία"
+
+#: ../vnc.html:196
+msgid "View Only"
+msgstr "Μόνο Θέαση"
+
+#: ../vnc.html:200
+msgid "Path:"
+msgstr "Διαδρομή:"
+
+#: ../vnc.html:204
+msgid "Scaling Mode:"
+msgstr "Λειτουργία Κλιμάκωσης:"
+
+#: ../vnc.html:206
+msgid "None"
+msgstr "Καμία"
+
+#: ../vnc.html:207
+msgid "Local Scaling"
+msgstr "Τοπική Κλιμάκωση"
+
+#: ../vnc.html:208
+msgid "Local Downscaling"
+msgstr "Τοπική Συρρίκνωση"
+
+#: ../vnc.html:209
+msgid "Remote Resizing"
+msgstr "Απομακρυσμένη Αλλαγή μεγέθους"
+
+#: ../vnc.html:213
+msgid "Repeater ID:"
+msgstr "Repeater ID:"
+
+#: ../vnc.html:219
+msgid "Style:"
+msgstr "Στυλ:"
+
+#: ../vnc.html:221
+msgid "default"
+msgstr "προεπιλεγμένο"
+
+#: ../vnc.html:227
+msgid "Logging:"
+msgstr "Καταγραφή:"
+
+#: ../vnc.html:234
+msgid "Apply"
+msgstr "Εφαρμογή"
+
+#: ../vnc.html:241 ../vnc.html:271
+#, fuzzy
+msgid "Connect"
+msgstr "Συνδέεται..."
```

Comment:
`fuzzy` means that it found something similar, but not a perfect match. These needs to be adjusted and the `fuzzy` comment removed.